### ISSUE TITLE
Dysfunctional COBOL import result for DISPLAY statements with LINE, COLUMN, and WITH 

### ISSUE THREAD

**Issue by codemanyak** [fesch/Structorizer.Desktop#1052]
COBOL import of certain DISPLAY statements with screen positioning (possibly spread over several lines) induce syntactically incorrect output statements in Structorizer:
```cobol
    Display space 
        line af-line 
        column 42 
        with size 30 
    end-display. 
```
produces:
![grafik](https://user-images.githubusercontent.com/15326471/182002192-f814a797-03cf-4713-ab37-d4f27635f957.png)

The insertion of "line, <line_no>" or "column, <column_no>" makes little sense in Structorizer, the conversion of "with size <size>" to "with_size_<size>" is also nonsense, the insertion of a comma for every new source line causes illegal empty expressions.

**Comment by codemanyak** [comment]
With direct screen positioning there is of course little adequate we could do in Structorizer. An idea might be to generate a procedure call like `DISPLAY_AT(' ', af_line, 42, 30)` ad to provide a standard subroutine diagram `DISPLAY_AT(expression, line, column, size)` or so, which explains the intended action in the head comment and then just contains an OUTPUT instruction for the passed-in expression, ignoring all positioning arguments.
The algorithmic challenge is, that the "at line/column" constuct is just one of many possible "display clauses" (where "with <display attr>" is another) that might follow the expression list, even in combination.
So either we must prepare a wild bunch of specific display procedures for any sensible combination or we just leave the emerging nonsense as is.

**Comment by codemanyak** [comment]
New approach now puts all display clauses into an additional comment line, thus resulting in a working instruction that does not lose information:
![grafik](https://user-images.githubusercontent.com/15326471/182015031-358848d5-2ccf-410c-aa8d-0964d6739241.png)

Fix will be delivered with version 3.32-09
